Casks/tweetbot.rb

```diff
@@ -10,8 +10,13 @@
 
   livecheck do
     url "https://tapbots.net/tweetbot4/update.plist"
-    strategy :extract_plist do |version|
-      "#{version.values.map(&:short_version).compact.first},#{version.values.map(&:version).compact.first}"
+    regex(%r{
+      <key>shortVersion</key>.*\n.*<string>(\d+(?:\.\d+)+)</string>
+      (?:.*\n){3}.*
+      <key>version</key>.*\n.*<integer>(\d+)</integer>
+    }ix)
```

Comment:
```suggestion
    regex(%r{
      <key>shortVersion</key>.*?<string>(\d+(?:\.\d+)+)</string>
      .*?
      <key>version</key>.*?<integer>(\d+)</integer>
    }imx)
```

We can simplify this a little if we use the multiline (`m`) flag and make the `.*` parts non-greedy (i.e., `.*?`).

Reply:
Sure! Would you prefer this as an additional commit or have the first commit amended?

Reply:
Since you agree with the changes, I'll just commit it and it'll get squashed by BrewTestBot when it's merged.